app/unit-tests/resources/commcare-apps/phone_call/default/app_strings.txt

```diff
@@ -0,0 +1,26 @@
+app.display.name=1Scratchwork
```

Comment:
Not positive but I think you can get rid of the app_strings resources and have the tests still work fine